### PR TITLE
Fix the num_batches value in error message

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

The warning was displaying the updated `num_batches`.

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [x] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

Part of #1

cc @borda @justusschock @awaelchli @ninginthecloud

### CODE REVIEW

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by awaelchli — APPROVED** [review]
great!

File: tests/trainer/test_data_loading.py [hunk 1]
```diff
@@ -339,6 +339,20 @@ def test_pre_made_batches():
...
+    limit_eval_batches."""
+    model = BoringModel()
+    dl_size = len(model.val_dataloader())
+    limit_val_batches = 1 / (dl_size + 2)
+    trainer = Trainer(limit_val_batches=limit_val_batches)
+    trainer._data_connector.attach_data(model)
+    with pytest.raises(
+        MisconfigurationException,
+        match=fr"{limit_val_batches} \* {dl_size} < 1. Please increase the `limit_val_batches`",
+    ):
```
Comment (rohitgr7) [inline tests/trainer/test_data_loading.py:353]:
```suggestion
    ):
        trainer._reset_eval_dataloader(RunningStage.VALIDATING, model)
```
